src/runtime_src/tools/scripts/pkg_hw_platform.sh

```diff
@@ -460,6 +460,8 @@ initXsaBinEnvAndVars()
          fwManagement="${XILINX_XRT}/share/fw/mgmt.bin"
       elif [ "${CardMgmtControllerFamily}" == "CMC-Gen1" ]; then
          fwManagement="${XILINX_XRT}/share/fw/cmc.bin"
+      elif [ "${CardMgmtControllerFamily}" == "CMC-NO-SC-GEN1" ]; then
+	 fwManagement="${XILINX_XRT}/share/fw/cmc_no_sc.bin"
```

Comment:
Earlier this week Martin P's commitment added some changes to pkgdsa.sh:
https://github.com/Xilinx/XRT/commit/acb9f868e99cb7fff78605f2d70454a9efe90cb5
which has yet to make it into pkg_hw_platform.sh.   

These changes are around adding additional CMC images with the new naming convention.

Could you either:
1) Work with Martin P to have him merge the changes into pkg_hw_platform.sh 
or
2) Merge the changes in yourself and update the flow for CMC-NoSC-Gen1

Reply:
Hi Stephen, I dropped a mail to Martin regd this. If my changes looks fine, please let it go into master branch as this change has to be available there by 26th Aug. Thanks.

Reply:
Hi Stephen, I dropped a mail to Martin regd this. If my changes looks fine, please let it go into master branch as this change has to be available there by 26th Aug. Thanks.

Reply:
Hello Raj, as indicated this would have been great if you could have merged his changes, but not necessary.  Please do keep me up-to-date with regards to your's and Martin's progress in this regarding this merge.

Reply:
Hi Stephen, I have added his changes also in this commit. I have not received any reply from him yet. I will keep you updated with latest. Thanks.